internal/remediation/override.go

```diff
@@ -128,13 +129,24 @@ func overridePatchVulns(ctx context.Context, cl client.ResolutionClient, result
 			seenVKs := make(map[resolve.VersionKey]struct{})
 			// Use the DependencyChains to find all the affected nodes.
 			for _, c := range v.ProblemChains {
+				// Currently, there is no way to know if a specific classifier or type exists for a given version with deps.dev.
```

Comment:
Can you add a TODO here to better handle this by checking against the Maven registry ?